src-ts/lib/modals/contact-support-modal/ContactSupportModal.tsx

```diff
@@ -0,0 +1,90 @@
+import { Dispatch, FC, SetStateAction, useContext, useState } from 'react'
+
+import { Form, FormDefinition, formGetInputModel, FormInputModel, formOnReset } from '../../form'
+import { profileContext, ProfileContextData } from '../../profile-provider'
+import { BaseModal } from '../base-modal'
+
+import { contactSupportFormDef, ContactSupportFormField } from './contact-support-form.config'
+import { ContactSupportRequest } from './contact-support-functions'
+import { contactSupportSubmitRequestAsync } from './contact-support-functions/contact-support-store'
+import styles from './ContactSupportModal.module.scss'
+
+export interface ContactSupportModal {
+    isOpen: boolean
+    onClose: () => void
+    workId?: string
+}
+
+const ContactSupportModal: FC<ContactSupportModal> = (props: ContactSupportModal) => {
+
+    const { profile }: ProfileContextData = useContext(profileContext)
+    const [formDef, setFormDef]: [FormDefinition, Dispatch<SetStateAction<FormDefinition>>] = useState<FormDefinition>({ ...contactSupportFormDef })
+
+    function generateRequest(inputs: ReadonlyArray<FormInputModel>): ContactSupportRequest {
+        const firstName: string = formGetInputModel(inputs, ContactSupportFormField.first).value as string
```

Comment:
I realize this may not be a new change, but it seems like it might read better as `getFormInputModel` rather than `formGetInputModel` -- is there a reason you chose the other?

Reply:
the function is named getInputModel in the form.functions scope. so when i export from a functions scope, i treat the name of the functions like a namespace and prefix it to the function names...

`export { getInputModel as formGetInputModel } from './form.functions.ts'`

Reply:
Fair enough, works for me.